render/gles2/renderer.c

```diff
@@ -323,7 +323,7 @@ static struct wlr_texture *gles2_texture_from_dmabuf(
 static void gles2_init_wl_display(struct wlr_renderer *wlr_renderer,
 		struct wl_display *wl_display) {
 	struct wlr_gles2_renderer *renderer =
-		gles2_get_renderer_in_context(wlr_renderer);
+		gles2_get_renderer(wlr_renderer);
```

Comment:
Hmm. We should probably switch context here.

Reply:
Tried one more time after your comment. If I revert this change I have the assert on startup.

Reply:
Ah - I probably took you wrong. You are telling about switching context and I was thinking about reverting this change to original function (which is asserting and not switching context, as this was your point regarding rendering functions)... Yeah - I can switch context here as well.